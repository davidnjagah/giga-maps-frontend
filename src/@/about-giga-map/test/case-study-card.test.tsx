import { describe, test, } from '@jest/globals';
import { render, } from '@testing-library/react';

import CaseStudyCard from '../common/case-study-card';

describe('CaseStudyCard', () => {
  test('renders CaseStudyCard and take a snapshop', () => {
    const { asFragment } = render(<CaseStudyCard cardPoster={''} title={''} description={''} column={0} />);
    expect(asFragment()).toMatchSnapshot();
  });
})