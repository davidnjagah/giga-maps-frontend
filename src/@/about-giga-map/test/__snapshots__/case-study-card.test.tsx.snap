// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaseStudyCard renders CaseStudyCard and take a snapshop 1`] = `
<DocumentFragment>
  .c0 {
  flex: 1 1 0;
  padding-left: 1rem;
  padding-right: 1rem;
  min-width: calc(100% / 0);
}

.c0 .right-spacing {
  flex: 1 1 0;
  margin-right: 1px;
}

.c1 {
  width: 100%;
  height: 18rem;
}

.c1 img {
  height: 100%;
  width: 100%;
  border-radius: 10px;
  object-fit: cover;
}

.c2 {
  height: max-content;
  width: 100%;
  padding: 1.5rem 1rem;
}

.c2 h3 {
  margin-top: 1rem;
  font-size: 2rem;
  font-weight: 500;
  line-height: 1.75rem;
  color: #fff;
}

.c2 p {
  margin-top: 3.5rem;
  font-size: 1.125rem;
  font-weight: 400;
  line-height: 1.6875rem;
  color: #fff;
}

@media (max-width:768px) {
  .c0 {
    padding-left: 0.3rem;
    padding-right: 0.3rem;
  }
}

@media (max-width:768px) {
  .c2 {
    padding: 2rem 0;
  }

  .c2 h3 {
    margin-top: 0;
    font-size: 1.25rem;
    line-height: 2rem;
    padding-bottom: 1rem;
  }

  .c2 p {
    margin-top: 0;
    font-size: .875rem;
    line-height: 1.25rem;
  }
}

<div
    class="c0"
    column="0"
  >
    <div
      class="right-spacing"
    >
      <div
        class="c1"
      >
        <img
          src=""
        />
      </div>
      <div
        class="c2"
      >
        <h3 />
        <p />
      </div>
    </div>
  </div>
</DocumentFragment>
`;
